src/stache/includes/cli/template.html

```diff
@@ -4,6 +4,9 @@
   <li>
     <sky-code>-t addin</sky-code> — References the <a href="https://github.com/blackbaud/skyux-sdk-template-addin">SKY UX Add-in template</a> to create <a href="https://developer.blackbaud.com/skyapi/docs/addins">SKY Add-ins</a> that allow developers to build features that integrate with Blackbaud solutions.
   </li>
+  <li>
+    <sky-code>-t barebones</sky-code> — References a pared down version of the <a href="https://github.com/blackbaud/skyux-sdk-template-barebones">SKY UX Add-in template</a> to create <a href="https://developer.blackbaud.com/skyapi/docs/addins">SKY Add-ins</a> that allow developers to build features that integrate with Blackbaud solutions. This template eliminates folders, modules, and references to packages that are not required to build add-ins.
```

Comment:
The text for the first link seems off to me. I mean, the link text is the same as in the `-t addin` description, but it points to a different repo. That seems a little confusing to me. It's a minor thing, but what do you think about making "pared down version of the" part of the link text? I think that makes it clearer what we are linking to.